src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -1231,7 +1230,10 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
                 oldColumn.name = newColumn.name
             }
 
-            if (this.isColumnChanged(oldColumn, newColumn, true, true)) {
+            if (
+                this.isColumnChanged(oldColumn, newColumn, true, true) ||
```

Comment:
Is there a reason why we can't add the `length` check to the `isColumnChanged` method?